Core/Utilities/Plotting/TypeLoader.py

```diff
@@ -16,21 +15,14 @@
 
 
 class TypeLoader(object):
```

Comment:
This was failing as it's a singleton despite having a cache for the `plottingFamily`. I don't see why it needs to be a singleton so I've removed it but if I'm missing something I could modify the cache to be dependent on the arguments.